draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -454,7 +436,42 @@ about individual client data.
 
 # Applying Privacy Partioning
 
-TODO: talk about how to think about new protocols that are using partitioning, the models that we derive from looking at the existing ones, how you recognize a protocol that's using this, and what tools do we have to evaluate it
+Applying privacy partitioning to an existing or new system or protocol requires the following steps:
+
+1. Identify the types of information used or exposed in a system or protocol, some of which can be used to identify a user or correlate to other contexts.
+1. Partition different types of user-identifying or correlatable information across contexts to minimize the amount of
+user-identifying or correlatable information in any single context.
+
+The most impactful types of user-identifying information to partition are (a) user identity or identities that can be linked and (b) user data.
```

Comment:
I'm not sure I follow you, since that's exactly what the text here says. That is, it says the most important things to separate are information relating to user identity (IP address) and user data (HTTP request).

Reply:
```suggestion
The most impactful partitioning is separating user-identifying information from other user data.
```

Reply:
it's not quite what the text says because it currently says that user data (b) is also user-identifying information which is confusing.

Reply:
User data _can_ be user-identifying, since it's data about or generated by the user. This is an important point I don't think we can omit. How about the following?

```
The most impactful types of information to partition are (a) user identity or identities that can be linked and (b) user data, which can be user-identifying for certain applications.
```

Reply:
I guess this sentence is not wrong but not sure it's the point. E.g. if I visit videosofcats.com with my IP address, someone who observes the traffic knows that I love cats. So separating this information is useful. If I visit myprivatepageofcatvideos.org where I'm the only user ever visiting this page, an observer can identify me no matter if they see my IP address or not. Hiding the IP address might still be important to not correlate this traffic with other traffic. But isn't the first case the most/more important one, where the user data in itself in not user-identifying?

Reply:
Yep, and that's an example of how this principle, even if applied correctly, is insufficient in many cases. This PR does acknowledge this limitation, so I don't think further changes are needed here.

Reply:
So why is “which can be user-identifying for certain applications” needed?

Reply:
isn't the most impactful partition to separate user-identifying data from non-user-identifying data?